src/containers/applications/tabs2.scss

```diff
@@ -54,27 +54,26 @@ rediv {
   display: flex;
   flex-direction: column;
   align-items: flex-end;
-  // justify-content: flex-end;
   padding: 0 32px;
-  color: #333;
+  color: var(--dark-txt);
```

Comment:
--med-txt

Reply:
#333 was when it was win10 one but in real win11 it is #fff

Reply:
okay then